onmt/translate/predictor.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
+""" Classifier Class and builder """
+from __future__ import print_function
+import codecs
+import time
+
+import torch
+import torchtext.data
+import onmt.model_builder
+import onmt.inputters as inputters
+from onmt.utils.misc import set_random_seed
+
+
+def build_classifier(opt, logger=None, out_file=None):
+    """Return a classifier with result redirect to `out_file`."""
+
+    if out_file is None:
+        out_file = codecs.open(opt.output, 'w+', 'utf-8')
```

Comment:
(a detail, but wasn't `w` enough here?)

Reply:
Ah, yes you're right. I just copied the same block in translator.py.